### PR TITLE
Fix console, add type annotations

### PR DESCRIPTION
**Pull request by palango**
Fixes: #5071

## Description

This was a leftover from the recent `ProxyManager` refactoring I did. 

This PR also adds type annotations, as mypy easily found that as soon as it started checking that file.

## PR review check list

Quality check list that cannot be automatically verified.

- Safety
    - [ ] The changes respect the necessary conditions for safety (https://raiden-network-specification.readthedocs.io/en/latest/smart_contracts.html#protocol-values-constraints)
-  Code quality
    - [ ] Error conditions are handled
    - [ ] Exceptions are propagated to the correct parent greenlet
    - [ ] Exceptions are correctly classified as recoverable or unrecoverable
- Compatibility
    - [ ] State changes are forward compatible
    - [ ] Transport messages are backwards and forward compatible
- Commits
    - [ ] Have good messages
    - [ ] Squashed unecessary commits
- If it's a bug fix:
    - Regression test for the bug
        - [ ] Properly covers the bug
        - [ ] If an integration test is used, it could not be written as a unit test
- Documentation
    - [ ] A new CLI flag was introduced, is there documentation that explains usage?
- Specs
    - [ ] If this is a protocol change, are the specs updated accordingly? If so, please link PR from the specs repo.
- Is it a user facing feature/bug fix?
    - [ ] Changelog entry has been added

### CODE REVIEW

File: raiden/ui/console.py [hunk 1]
```diff
@@ -147,33 +158,34 @@ def lasterr(n=1):
 
 
 class ConsoleTools:
-    def __init__(self, raiden_service, settle_timeout):
-        self._chain = raiden_service.chain
```
Comment (palango) [inline raiden/ui/console.py]: This is the bug fix

File: raiden/ui/console.py [hunk 2]
```diff
@@ -45,19 +56,19 @@ def print_usage():
...
 
-def inputhook_gevent():
+def inputhook_gevent() -> int:
     while not stdin_ready():
         gevent.sleep(0.05)
     return 0
 
 
 @inputhook_manager.register("gevent")
 class GeventInputHook:
```
Comment (rakanalh) [inline raiden/ui/console.py]: This whole class seems to be unused. Can you remove it please?

Comment (palango) [reply raiden/ui/console.py]: I thought the same, but as it gets registered I wasn't sure that this isn't used somehow.

**Comment by palango** [comment]
@rakanalh I prefer to leave it in for now as I don't fully understand the implications of removing it. However, I opened https://github.com/raiden-network/raiden/issues/5082 to track that issue

**Pull request merged**